Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla48236.cs

```diff
@@ -20,6 +14,10 @@ public class Bugzilla48236 : TestContentPage
 	{
 		protected override void Init()
 		{
+			// Refer to https://bugzilla.xamarin.com/show_bug.cgi?id=48236 for this issue. WinRT/UWP had
+			// an issue where the background color would be used on the area containing the buttons, 
+			// potentially causing the color to run the width of the screen. Only the buttons should have
+			// a background color.
```

Comment:
The comments are good, but I meant on-screen text for anyone manually running the test. So they have some idea of what constitutes success and failure (e.g., the on-screen text for [36014](https://github.com/xamarin/Xamarin.Forms/blob/74cb5c4a97dcb123eb471f6b1dffa1267d0305aa/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla36014.cs) ).